go/runtime/scheduling/simple/simple.go

```diff
@@ -16,14 +21,23 @@ const (
 )
 
 type scheduler struct {
-	incomingQueue *incomingQueue
+	logger *logging.Logger
 
-	dispatcher api.TransactionDispatcher
+	maxQueueSize uint64
```

Comment:
```suggestion
	maxTxPoolSize uint64
```
Also consider grouping it together with the tx pool.